### PR TITLE
make compatible with babel 6, bump version to 2.0.0

### PR DESCRIPTION
**Pull request by rossipedia**
Not much more to talk about here :)

### CODE REVIEW

**Comment by chandlerprall** [comment]
Thanks for putting this together. I'll verify it tomorrow; I see that the tests didn't change at all, are they still valid?

**Comment by rossipedia** [comment]
Good catch. They're broken. I'll work on fixing those up tonight and update the PR.

**Comment by rossipedia** [comment]
All tests passing now

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "babel-plugin-jsx-display-if",
-  "version": "1.0.2",
+  "version": "2.0.0",
   "description": "babel plugin to add conditional display ala ng-if to jsx",
   "main": "dist/index.js",
   "scripts": {
     "test": "mocha --compilers js:babel-core/register --reporter spec",
-    "build": "babel -d dist index.js"
+    "build": "node_modules/.bin/babel -d dist index.js"
```
Comment (donabrams) [inline package.json:8]: All npm package.json scripts should be executed with node_modules/.bin in path

**Comment by donabrams** [comment]
Awesome PR, thanks @rossipedia!

**Comment by rossipedia** [comment]
Anytime, awesome little lib :+1:

**Pull request merged**